### PR TITLE
6071 Make correlation attrs for account arts (part one)

### PR DESCRIPTION
**Pull request by rcordovano**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java [hunk 1]
```diff
@@ -48,55 +48,61 @@ public static String getEmailAddressAttrString() {
...
-     * @return List of EamArtifacts
+     * @param artifact An artifact.
+     *
+     * @return A list, possibly empty, of correlation attribute instances for
+     *         the artifact.
      */
-    public static List<CorrelationAttributeInstance> makeInstancesFromBlackboardArtifact(BlackboardArtifact artifact,
-            boolean checkEnabled) {
-        List<CorrelationAttributeInstance> eamArtifacts = new ArrayList<>();
+    public static List<CorrelationAttributeInstance> makeAttrsForArtifact(BlackboardArtifact artifact) {
```
Comment (raman-bt) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java]: Terms Artifact & Attrs are used throughout our code and easy to confuse.  I think explicitly naming the method makeCorrAttrsForArtifact() would be useful to clarify this method makes Correlation Attribute (instances) as opposed to TSK attributes.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*
  * Central Repository
  *
- * Copyright 2015-2020 Basis Technology Corp.
+ * Copyright 2017-2020 Basis Technology Corp.
```
Comment (raman-bt) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java]: Why does the Copyright start date change here?

File: Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/IngestEventsListener.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*
  * Central Repository
  *
- * Copyright 2015-2019 Basis Technology Corp.
+ * Copyright 2017-2020 Basis Technology Corp.
```
Comment (raman-bt) [inline Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/IngestEventsListener.java]: Again, why is start date changing?

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java (hunk 1 above)
Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java]: That's a good idea. I will make the change. Please stand by for a new commit.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java (hunk 2 above)
Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java]: The copyright notice was copied and pasted into this file without regard for the actual date in the notice. The initial work on the feature we now know as the central repository was done in 2017. This change was part of my ongoing campaign to make the copyright notices more accurate.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java (hunk 1 above)
Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java]: That's a good idea. I will make the change. Please stand by for a new commit.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java (hunk 2 above)
Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java]: The copyright notice was copied and pasted into this file without regard for the actual date in the notice. The initial work on the feature we now know as the central repository was done in 2017. This change was part of my ongoing campaign to make the copyright notices more accurate.

File: Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/IngestEventsListener.java (hunk 3 above)
Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/IngestEventsListener.java]: See previous answer.

**Pull request merged**